docs/quick-start/index.md

```diff
@@ -682,19 +685,240 @@ public Server(String serverName) {
+client-server communications before closing the client. 
+
+Now, let's review the main thing the `Client` class does, sending the `Print` command.
+
+### Sending the command
+<?embed-code file="examples/hello/src/main/java/io/spine/helloworld/client/Client.java" 
+             start="sendCommand() {" 
+             end="    }"?>
+```java
+public void sendCommand() {
+    String userName = System.getProperty("user.name");
+    Print commandMessage = Print.newBuilder()
+            .setUsername(userName)
+            .setText("Hello World!")
+            .vBuild();
+    this.subscriptions =
+            client.asGuest()
+                  .command(commandMessage)
+                  .observe(Printed.class, this::onPrinted)
+                  .post();
+}
+```
+
+The method does two things:
+   1. Creates a command message using the login name of the current computer user.
+   2. Posts the command, subscribing to the `Printed` event which will be generated as the result
+      of the handling the posted command.
+      
+Let's review the second thing in details.
+   
+The `client.asGuest()` call starts composing a client request. For the sake of the simplicity,
+we on do this on behalf of a user who is not logged in. A real-world application would send
+start sending most of the commands using the `client.onBehalfOf(UserId)` call.
+
+The `command(commandMessage)` call tells we want to send a command with the passed message.
+
+Then, we subscribe to the `Printed` event which would be generated as the result of handling
+the command. The events obtained from the server would be passed to the `onPrinted()` method
+of the `Client` class.
+
+The `post()` method sends the command to the server, returning the set with one `Subscription`
+to the `Printed` event. We store the returned set in the field, so that it can be used later 
+for cancelling the subscription upon receiving the event.  
+
+### Handling the event
+
+When the client code receives the `Printed` event, it prints its data and then 
+cancels the subscription:
+
+<?embed-code file="examples/hello/src/main/java/io/spine/helloworld/client/Client.java" 
+             start="void onPrinted(" 
+             end="    }"?>
+```java
+private void onPrinted(Printed event) {
+    printEvent(event);
+    cancelSubscriptions();
+}
+```
+
+There's not much exciting about the printing part.
+<?embed-code file="examples/hello/src/main/java/io/spine/helloworld/client/Client.java" 
+             start="void onPrinted(" 
+             end="    }"?>
+```java
+private void onPrinted(Printed event) {
+    printEvent(event);
+    cancelSubscriptions();
+}
+```
+The only interesting thing here is the statically imported `Json.toCompactJson()` call which 
+converts the event message to a `String`. The method is available from the `Json` utility class
+provided by the framework. The utility does heavy lifting of the conversion which involves knowing
+all Protobuf types available in the project.
```

Comment:
I want to keep this piece because here we put a small “hook” for learning that printing to JSON or text isn't an easy thing. That's why we speak about “heavy lifting”. This is for people who struggled with using JSON in their projects. A user interested in the API of the utility will find it easily.